README.md

```diff
@@ -197,6 +198,20 @@ go_library(name, srcs, deps, data)
         <p>List of files needed by this rule at runtime.</p>
       </td>
     </tr>
+    <tr>
```

Comment:
One question: different from ".c" or ".h", the "go tool asm" command has to be run separately for each ".s" file which generates multiple ".o" files. I had the problem to define a proper outputs attr for these actions.

Thanks.